dbt_af/integrations/af_callbacks/__init__.py

```diff
@@ -0,0 +1,5 @@
+from .callbacks import prepare_callbacks  # noqa
```

Comment:
To give a more meaningful name, we can name it `af_custom_callbacks` to show that it could be an arbitrary set of callbacks with integrations with multiple systems.

Reply:
function was renamed to `af_custom_callbacks`